infra/docker/jupyter/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# Feast Jupyter Server
+# Entrypoint Script
+#
+
+# Update permissions to allow notebook user access to - Google SA key 
```

Comment:
Isn't the NB_UID always the same? Could we statically provide access to this service account to the user by granting ownership ahead of time?

Reply:
Its possible, we can grant permissions by `chown` before `docker-compose up`:
```sh
chown 1000:1000 infra/docker-compose/gcp-service-accounts/key.json
docker-compose up
```
closing this PR in favour of docs to direct users to do this.